scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -265,6 +265,9 @@ <h4 class="">
         {% if case.vcf_files.vcf_cancer_sv %}
           <a class="btn btn-dark btn-sm text-white" href="{{ url_for('variants.cancer_sv_variants', institute_id=institute._id, case_name=case.display_name, variant_type='clinical') }}">Clinical {% if case.track=="cancer" %}somatic {% endif %}structural variants</a>
         {% endif %}
+        {% if case.vcf_files.vcf_fusion %}
```

Comment:
Would be nice to have a direct link from caseS page, that now looks like this:

<img width="806" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/045ad4d4-7839-4199-a5d4-37d89cd3bc4a">